samtranslator/open_api/open_api.py

```diff
@@ -210,7 +210,7 @@ def add_lambda_integration(
         path_dict[method][self._X_APIGW_INTEGRATION] = {
             "type": "aws_proxy",
             "httpMethod": "POST",
-            "payloadFormatVersion": "1.0",
+            "payloadFormatVersion": "2.0",
```

Comment:
Does this mean we default to 2.0 payloadFormatVersion in SAM?

Reply:
Ahh.. yes, as stated in the docs below.

Reply:
Yes, the default value is "2.0" in SAM